pkg/chartverifier/checks/charttesting_test.go

```diff
@@ -154,25 +139,25 @@ func TestVersionSetting(t *testing.T) {
 	testCases := []testCase{
 		{
 			description: "oc.Version returns 4.7.9",
-			holder:      &testAnnotationHolder{},
+			opts:        &CheckOptions{AnnotationHolder: &testAnnotationHolder{}},
 			versioner:   getVersionGood,
 			version:     "4.7.9",
 		},
 		{
 			description: "oc.Version returns error, flag set to 4.7.8",
-			holder:      &testAnnotationHolder{CertifiedOpenShiftVersionFlag: "4.7.8"},
+			opts:        &CheckOptions{AnnotationHolder: &testAnnotationHolder{CertifiedOpenShiftVersionFlag: "4.7.8"}},
```

Comment:
Is this not a different name now? TestedOpenShiftVersion?

Reply:
Yes, though this is a variable name, not the actual value. Seems like the `CertifiedOpenShiftVersion` is still being used in other places of the program.